src/typescript/sdk/types/order.d.ts

```diff
@@ -0,0 +1,61 @@
+import { type BCS } from "aptos";
+
+// Order Side
+export enum Side {
+  Ask = 0,
+  Bid = 1,
+}
```

Comment:
It's better to use string unions rather than numeric enums, because TypeScript type checking doesn't work well with numeric enums. For example, it won't complain if instead of

```ts
txBuilder.placeLimitOrderUserEntry(
  "0x1::aptos_coin::AptosCoin",
  "0x2::test_usdc::TestUSDCoin",
  0n,
  "0x3",
  Side.Bid,
  1000n,
  1000n,
  Restriction.ImmediateOrCancel,
  SelfMatchBehavior.Abort
);
```

You enter

```ts
txBuilder.placeLimitOrderUserEntry(
  "0x1::aptos_coin::AptosCoin",
  "0x2::test_usdc::TestUSDCoin",
  0n,
  "0x3",
  5
  1000n,
  1000n,
  Restriction.ImmediateOrCancel,
  SelfMatchBehavior.Abort
);
```